sycl/test/check_device_code/esimd/dae.cpp

```diff
@@ -15,10 +15,9 @@ __attribute__((sycl_kernel)) void my_kernel(Func kernelFunc) {
 
 SYCL_EXTERNAL SYCL_ESIMD_FUNCTION ESIMD_NOINLINE void callee(int x) {}
 
-// CHECK: define dso_local spir_kernel {{.*}} !kernel_arg_addr_space ![[#MD:]]
-// CHECK: !kernel_arg_access_qual ![[#MD]] !kernel_arg_type ![[#MD]] !kernel_arg_base_type ![[#MD]] !kernel_arg_type_qual ![[#MD]] !kernel_arg_accessor_ptr ![[#MD]]
+// CHECK: define dso_local spir_kernel {{.*}} !sycl_kernel_omit_args ![[#MD:]]
```

Comment:
@intel/dpcpp-esimd-reviewers, I'm sure I understand the purpose of this test, but I tried to check that "dead argument elimination" optimization attaches metadata correctly.
It looks like original test verifies that other metadata do not refer to deleted argument. Should I check for `!kernel_arg_accessor_ptr`?

Reply:
your change looks fine thx